shared/util/forward-logs.desktop.js

```diff
@@ -112,14 +112,23 @@ function setupSource () {
   }
 
   ['log', 'warn', 'error'].forEach(key => {
+    if (__DEV__ && typeof window !== 'undefined') {
+      window.console[`${key}_original`] = window.console[key]
```

Comment:
helps debug console issues, we can get the raw dump and not the one that plumbs through to the main thread